gravity-forms/gw-populate-date.php

```diff
@@ -1418,6 +1420,18 @@ function processTzCorrection(tzOffset, oldValue) {
 								result.setFullYear(this.y, this.m, this.d);
 								result.setHours(this.h, this.i, this.s, this.f);
 
+								if ( this.rwd ) {
+									var mod = this.rwd < 0 ? -1 : 1;
```

Comment:
What does the `r` in this stand for?

Also, what is this line doing specifically?

Reply:
Haha, I have no idea. Just followed suit with the naming pattern of all the other variables of this type.

I added a comment to provide some clarity. Essentially, we need to add or subtract a day for each weekday allowance specified in the `this.rwd` variable. We know it should subtract if `this.rwd` is a negative number. Below, this `mod` variable is used to automatically handle whether a day is added or subtracted.

Reply:
Nice, thank ya!